src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,29 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    private static int salary;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[] dateToSplit = dateTo.split("[.]");
+        String[] dateFromSplit = dateFrom.split("[.]");
+        int dateToKod = Integer.parseInt(dateToSplit[2] + dateToSplit[1] + dateToSplit[0]);
+        int dateFromKod = Integer.parseInt(dateFromSplit[2] + dateFromSplit[1] + dateFromSplit[0]);
```

Comment:
Use LocalDate and Formatter
![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/0e050c48-ad88-40e3-9514-3c69d30f8a7b)